core/src/main/java/nl/nn/adapterframework/webcontrol/api/Webservices.java

```diff
@@ -86,6 +86,7 @@ public Response getWebServices() throws ApiException {
 					if (rl.isView()) {
 						Map<String, Object> service = new HashMap<String, Object>(2);
 						service.put("name", adapter.getName() + " "+  receiver.getName());
+						service.put("method", rl.getMethod());
 						service.put("uriPattern", rl.getUriPattern());
 						webServices.add(service);
 					}
```

Comment:
Does this only show the 'vieawable' restlisteners?

Reply:
yes

Reply:
This doesn't add all restlisteners to the GUI, only the method (which is always `GET` for listeners with isView).
I suggest removing the isView or adding it as attribute.